.pre-commit-config.yaml

```diff
@@ -102,15 +102,11 @@ repos:
             - black
             - isort
 
--   repo: local
+-   repo: https://github.com/pre-commit/mirrors-clang-format
+    rev: fdd6cd7821ce22929625f134bc97488ed699e256 # frozen: 12.0.1
     hooks:
-    - id: clang-format
-      name: Run clang-format
-      exclude: src/deps
-      entry: clang-format -i -style=file
-      types: [text]
-      files: '.*\.(cpp|h|hpp|cu|cuh)$'
-      language: system
+    -   id: clang-format
+        types_or: [c++, c, c#, cuda] # also run on CUDA
```

Comment:
c# ? :partying_face:

Reply:
It's supported, so why not. Main issue is that the OG hook doesn't support CUDA as a type, although I am working on getting it fixed. 

Other supported languages include JSN btw.

Reply:
The CUDA type_or is now merged in, but won't be updated until a new release is made of the hook.